### DISCUSSION TITLE
Hello, World!

### DISCUSSION THREAD

**Discussion by MilesWK** [community/community#109372] [Other Feature Feedback, Questions, & Ideas]
Since I just joined the community, I thought I would say hi, so....

### 👋 Hi!
I am MilesWK. I have been programming for a LONG time now. First, I started off at [Scratch](https://scratch.mit.edu/). Actually, I am still there today, making animations and projects. Next, I went to [Replit](https://replit.com/). I actually quite liked the community, and the options. Then 2024 happened. At the start of 2024 Replit made a LOT of changes: comments on projects gone, no creditcardless deployments and so on.  I didn't like the changes so I moved to [Glitch](https://glitch.com/) where I am now. 

### Links

Here are some links:

- My portfolio website: [https://mileswk.glitch.me/](https://mileswk.glitch.me/)
- My scratch page: [https://scratch.mit.edu/users/MilesWK](https://scratch.mit.edu/users/MilesWK)
- My glitch page: [https://glitch.com/@MilesWK/](https://glitch.com/@MilesWK/)
- My favorite game that I made: [World's Hardest Game:](https://worlds-hardest-game.glitch.me/)
- Random quote website I made: [Random Quotes](https://random-quotes-mileswk.glitch.me/)

Have a great day!

**Comment by ghost** [comment]
Hey Miles these websites are really nice, looking more like this from you...

**Comment by queenofcorgis** [comment]
👋 Welcome to the GitHub Community on Discussions, @MilesWK  Happy to have you here 🙂

To help keep our space clutter-free, we recommend introducing yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204) and checking out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776). This post will be closed and locked to keep our community nice and tidy 🧹.
